website/docs/docs/partnerships.mdx

```diff
@@ -9,38 +9,30 @@ description: >-
-- **SSO (Single Sign On):** These partners focus on authentication for end users to securely sign on.
-- **CI/CD:** These partners focus on continuous integration and continuous delivery/deployment.
-- **VCS:** These partners focus on tracking and managing software code changes.
+- **Code Scanning:** Partners that provide tooling to review IaC configurations to prevent errors or security issues.
+- **Cost Management:** Partners that analyze the impact of new infrastructure to cost, and apply cost governance.
+- **Observability/Monitoring:** Partners focused on providing visibility of performance  or automatically detect infrastructure changes and ensure optimal observability in place.
+- **Security:** Partners that detect Terraform configuration errors that are misaligned with security and compliance policies.
+- **Audit:** Partners that help manage multi-cloud infrastructure resources to discover, prevent service disruptions, improve governance, and increase efficiency.
+- No code/low code: Partners focused on implementing, deployment and delivery of IT, Supply Chain, Operations Management, Business Management and other workflows
+- **SSO (Single Sign On):** Partners focused on authentication for end users to securely sign on.
+- **CI/CD:** Partners focused on continuous integration and continuous delivery/deployment.
+- **VCS:** Partners focused on tracking and managing software code changes.
 
 Most workflow partners create [run tasks](/cloud-docs/integrations/run-tasks) that let them integrate their services directly into the Terraform workflow. [Run tasks](/cloud-docs/integrations/run-tasks) let Terraform Cloud execute tasks in external systems at certain points in the Terraform Cloud run lifecycle. Specifically, Terraform Cloud users can configure run tasks to execute during the post-plan and the pre-plan (public beta) run stages. Refer to the [Terraform Cloud Integrations](/docs/partnerships#terraform-cloud-integrations) or [Terraform Enterprise Integrations](/docs/partnerships#terraform-enterprise-integrations) documentation for more details.
 
+Most workflow partners create run tasks that let them integrate their services directly into the Terraform workflow. Most workflow partners integrate with the Terraform workflow itself. Run tasks allow Terraform Cloud to execute tasks in external systems at specific points in the Terraform Cloud run lifecycle. This offers much more extensibility to Terraform Cloud customers, enabling them to integrate your services into the Terraform Cloud workflow. This feature allows users to add and execute these tasks during the post-plan and the new pre-plan & pre-apply stages. Eventually, we will open the third phase of this workflow to Terraform Cloud users, the post-apply stage. Reference the [Terraform Cloud Integrations](/docs/partnerships#terraform-cloud-integrations) or [Terraform Enterprise Integrations](/docs/partnerships#terraform-enterprise-integrations) documentation for more details.
```

Comment:
```suggestion
Most workflow partners create run tasks that let them integrate their services directly into the Terraform workflow. Most workflow partners integrate with the Terraform workflow itself. Run tasks allow Terraform Cloud to execute tasks in external systems at specific points in the Terraform Cloud run lifecycle. This integration offers much more extensibility to Terraform Cloud customers, enabling them to integrate your services into the Terraform Cloud workflow. This feature allows users to add and execute these tasks during the post-plan and the new pre-plan and pre-apply stages. Eventually, we will open the third phase of this workflow to Terraform Cloud users, the post-apply stage. Refer to the [Terraform Cloud Integrations](/docs/partnerships#terraform-cloud-integrations) or [Terraform Enterprise Integrations](/docs/partnerships#terraform-enterprise-integrations) documentation for more details.
```
Suggest adding "integration" to avoid a "This" without a subject.